### PR TITLE
README upgrades

### PR DESCRIPTION
**Pull request by jason-curtis**
- clearer instructions
- pip install actually works and is --quiet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,23 +25,27 @@ Examples of code that should probably *not* live here:
...
-Jupyter should be able to tab-complete all of our public functions once you run `import osmo_jupyter` and then type `osmo_jupyter.` and `<tab>`.
+3. Once you run the cell you created above, Jupyter will be able to tab-complete all of our public functions: type `osmo_jupyter.` and `<tab>`.
 
 
 ## Contributing
 
 ### Feature requests
 
-The software team takes feature requests and pull requests! Just talk to one of us and/or file a JIRA ticket in the Feature Requests epic.
+The software team takes feature requests and pull requests! Just talk to one of us and/or file a JIRA ticket in the [Feature Requests epic](https://osmobot.atlassian.net/browse/SOFT-306). Extra points for example code and clear explanations of what you're trying to accomplish.
```
Comment (dubchuck) [inline README.md:48]: Should we inform users to add a tag or another label so that feature requests are clearly for notebooks?

File: README.md [hunk 2]
```diff
@@ -51,7 +55,7 @@ The "bar" for including code in this library is higher than that seen in our jup
 
 ### Code Style
 
-This repo follows the Osmo [Code Style Manifesto](https://docs.google.com/document/d/1W1Ipug8IACL4PfZAq5bQKlmfcJGmHGKNH95_FwJcjaI).
+This repo follows the Osmo [Code Style Manifesto](https://docs.google.com/document/d/1W1Ipug8IACL4PfZAq5bQKlmfcJGmHGKNH95_FwJcjaI) as applies to Python code.
```
Comment (dubchuck) [inline README.md:58]: Perhaps a hashed link that jumps to the Python section?  I can see the value of needing to look at the overview before getting there.

**Review by dubchuck — APPROVED** [review]
A few questions but overall looks good!

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# This sets default codeowners for the repo. These people get automatically requested for review
+# See https://help.github.com/articles/about-codeowners/
+*       @OsmoSystems/software
```
Comment (jaimemarijke) [inline .github/CODEOWNERS:3]: Did you also add the restriction that commits to master must be pull requests? And that merges require approval from a codeowner?

**Review by jaimemarijke — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (jason-curtis) [reply README.md:58]: there isn't really just one python section that covers everything. a bunch of the general stuff applies too.

File: README.md (hunk 1 above)
Comment (jason-curtis) [reply README.md:48]: let's see if there's any confusion first

File: .github/CODEOWNERS (hunk 3 above)
Comment (jason-curtis) [reply .github/CODEOWNERS:3]: thanks for the reminder, just did it

**Pull request merged**